feat(maven-use): add support for the parent tag (#1147)
canonical/craft-parts: docs/reference/changelog.rst

New features:

- Make the error message more detailed and traceable when the maven-use plugin
  encounters invalid XML in the software's :file:`pom.xml` file.

> - Add support for the `parent` tag on a :file:`pom.xml` file with the maven-use plugin.

Reviewer: ```suggestion
- Add support for the ``parent`` tag in the :file:`pom.xml` file with the maven-use plugin.
```

There's only one, right?
